### ISSUE TITLE
Rebuild site whenever relevant files are changed in cookbook repositories

### ISSUE THREAD

**Issue by jukent** [ProjectPythia/cookbook-gallery-sphinx#81]
Whenever the thumbnail or `_config.py` files are changed in a relevant cookbook, we want the gallery to rebuild to display that cookbook's gallery card appropriately.

Some possible resources:
https://github.community/t/triggering-by-other-repository/16163
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

**Comment by erogluorhan** [comment]
@jukent do you think we should still pursue this in the newer MyST gallery? A couple things to note:

- Cookbooks no longer have `config.py` but changes on thumbnails etc. still applies
-  Implementing this vs maybe publishing site weekly might be a question to answer (Especially if the former will take too much time)

**Comment by jukent** [comment]
I think we could still pursue this, but it feels like a lesser priority compared to some of our other tasks (having the build fail if the notebook cells can't execute). I think the site should be set to re-build daily or weekly (and we should remember to manually trigger a re-build before events).

**Comment by erogluorhan** [comment]
Yeah, that all makes sense to me. Ok, I'll open issue in the new repo and get this one closed.

**Comment by erogluorhan** [comment]
Will be pursued in the new MyST gallery repo: https://github.com/ProjectPythia/cookbook-gallery/issues/23
